frontend/src/app.js

```diff
@@ -0,0 +1,48 @@
+
```

Comment:
Yeah, kind of want to see this moved back to `frontend/src/app/index.js` - if only to see a clearer diff in changes with the old react-router powered version